TOOLS/osxbundle/mpv.app/Contents/Info.plist

```diff
@@ -796,13 +993,91 @@
         <key>UTTypeIdentifier</key>
         <string>io.mpv.sub</string>
         <key>UTTypeReferenceURL</key>
-        <string>http://en.wikipedia.org/wiki/MicroDVD</string>
+        <string>https://en.wikipedia.org/wiki/MicroDVD</string>
         <key>UTTypeTagSpecification</key>
         <dict>
           <key>public.filename-extension</key>
           <array>
             <string>sub</string>
           </array>
+          <key>public.mime-type</key>
+          <array>
+            <string>text/plain</string>
+          </array>
+        </dict>
+      </dict>
+      <dict>
+        <key>UTTypeConformsTo</key>
+        <array>
+          <string>public.plain-text</string>
+        </array>
+        <key>UTTypeDescription</key>
+        <string>AQTitle Subtitle</string>
+        <key>UTTypeIconFile</key>
+        <string>document.icns</string>
+        <key>UTTypeIdentifier</key>
+        <string>io.mpv.aqt</string>
+        <key>UTTypeReferenceURL</key>
+        <string>https://web.archive.org/web/20070210095721/http://www.volny.cz/aberka/czech/aqt.html</string>
+        <key>UTTypeTagSpecification</key>
+        <dict>
+          <key>public.filename-extension</key>
+          <array>
+            <string>aqt</string>
+          </array>
+          <key>public.mime-type</key>
+          <array>
+            <string>text/plain</string>
+          </array>
+        </dict>
+      </dict>
+      <dict>
+        <key>UTTypeConformsTo</key>
+        <array>
+          <string>public.plain-text</string>
+        </array>
+        <key>UTTypeDescription</key>
+        <string>JACOSub Subtitle</string>
+        <key>UTTypeIconFile</key>
+        <string>document.icns</string>
+        <key>UTTypeIdentifier</key>
+        <string>io.mpv.jss</string>
+        <key>UTTypeReferenceURL</key>
+        <string>http://unicorn.us.com/jacosub/jscripts.html</string>
+        <key>UTTypeTagSpecification</key>
+        <dict>
+          <key>public.filename-extension</key>
+          <array>
+            <string>jss</string>
+          </array>
+          <key>public.mime-type</key>
+          <array>
+            <string>text/plain</string>
+          </array>
+        </dict>
+      </dict>
+      <dict>
+        <key>UTTypeConformsTo</key>
+        <array>
+          <string>public.plain-text</string>
+        </array>
+        <key>UTTypeDescription</key>
+        <string>RealText Subtitle</string>
+        <key>UTTypeIconFile</key>
+        <string>document.icns</string>
+        <key>UTTypeIdentifier</key>
+        <string>io.mpv.rt</string>
+        <key>UTTypeReferenceURL</key>
```

Comment:
i just tried to build a new bundle and it was broken, same for the nightly bundles from the pipeline.

this must be the reason, when you removed the URL you forgot to remove the key for it too.